### ISSUE TITLE
Error Detected Noinput on the remote server

### ISSUE THREAD

**Issue by iRioxe** [alphacep/unimrcp#4]
Hi!
I'm trying to use unimrcp-vosk-plugin and having error on the server side:
`2022-02-06 00:17:23:382691 [INFO]   Detected Noinput <1f025cda86e211ec@speechrecog>`

Server-machine configurations are good in case of open ports and etc, as I have a connection messages between the server and the client. I'm working with:
- the default repo code;
- examples from .../unimrcp/data folder;
- default image from the docker-hub;

The issue happens while remote usage only. If the client and the server beside the same local network everything works.

I've already tried to set additional time-out parameters, because thought may be this is the long response issue, but it seems like it's not the case. Ive used:
- No-Input-Timeout: 15000
- Recognition-Timeout: 20000
- Speech-Complete-Timeout: 9000
- DTMF-Term-Timeout: 3000
- DTMF-Interdigit-Timeout: 9500

My docker port settings are:
- udp,tcp: 8060,5060
- udp: 8062, 4000-5000, 5000-6000
- tcp: 1544,1554,8000,4900

I've used the default configuration parameters but interchanged with my server public IP only.
And docker launch command is:
`docker run -p 5000-6000:5000-6000/udp -p 1544:1544/tcp -p 1554:1554/tcp -p 8060:8060/udp -p 8060:8060/tcp -p 8062:8062/udp -p 4000-5000:4000-5000/udp -p 8000:8000/tcp -p 5060:5060/udp -p 4900:4900/tcp -p 554:554/tcp alphacep/vosk-unimrcp`

Would you mind to tell, are there any options to overcome this? And how to do it in details?
Thx, in advance

**Comment by nshmyrev** [comment]
It is about port forwarding, not about timeouts. You have to figure out which port is blocked, most likely RTP (5000).

**Comment by iRioxe** [comment]
Thx, for a fast response. I've made the ranges smaller (like 4000 to 4004, etc.) and checked them out:
- remote server `iptables -L -n` says that all ports have accept status
- localhost `nmap` output is the same as the remote server one, i.e. 554,544,8060 is opened and others are closed while server is running. As I understood It's just the way it works. I did such launches independetly;
- All my remote server ports are opened; I have an access and applyied short test with launching httpt.python website on each port and eache website is accesible via public web;

And have the same error.
Correct me, pls, if am doing smth wrong
